astericks/astericks.c++

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+using namespace std;
+
+int main(){
+
+    int rows{};

```

Comment:
why the {}?

Reply:
It is to initialize my variable;
This is called "Value initialization" and is used because I wanted to replace the value afterward. 
By this syntax the C++ will automatically assign 0 or a more proper value to the variable depending on the type, 
Therefor undefined behaviors by uninitialized variables are avoided.